app/src/views/register.ts

```diff
@@ -3,9 +3,13 @@ import PublicView from './public/';
 import SharedView from './shared/shared-view.vue';
 
 const PrivateView = defineAsyncComponent(() => import('./private'));
+const DrawerItem = defineAsyncComponent(() => import('./private/components/drawer-item.vue'));
+const DrawerBatch = defineAsyncComponent(() => import('./private/components/drawer-batch.vue'));
 
 export function registerViews(app: App): void {
 	app.component('PublicView', PublicView);
 	app.component('PrivateView', PrivateView);
 	app.component('SharedView', SharedView);
+	app.component('DrawerItem', DrawerItem);
+	app.component('DrawerBatch', DrawerBatch);
```

Comment:
I think we might want to register these under `/app/src/components/register.ts` for consistency's sake 👍🏻